### PR TITLE
Add list Freesound examples CharField for easy Admin edition

### PR DESCRIPTION
**Pull request by xavierfav**
In order to allow the modification from the Admin page of the examples of Freesound clips for each `TaxonomyNode`  (`ManyToManyField` to `Sound`), a `CharField` is created for manually entering a list of Freesound IDs.

The two fields (ManyToMany and CharField) are `synchronized` by overriding the _save()_ `TaxonomyNode` method so that any change in one of the field is passed to the other.

### CODE REVIEW

File: datasets/models.py [hunk 1]
```diff
@@ -179,10 +182,41 @@ class TaxonomyNode(models.Model):
...
+                        self.freesound_examples_verification.add(sound)
+                    except ObjectDoesNotExist:
+                        pass
+            if old.freesound_examples != self.freesound_examples:
+                self.list_freesound_examples = ','.join(
+                    [str(fsid) for fsid in list(self.freesound_examples.values_list('freesound_id', flat=True))])
+            if old.freesound_examples_verification != self.freesound_examples_verification:
+                self.list_freesound_examples_verification = ','.join(
+                    [str(fsid) for fsid in list(self.freesound_examples_verification.values_list('freesound_id', flat=True))])
+        models.Model.save(self, *args, **kwargs)
```
Comment (ffont) [inline datasets/models.py]: I think you should call save from the superclass, something like  `super(...)`

File: datasets/models.py [hunk 2]
```diff
@@ -498,7 +532,7 @@ def save(self, request=False, *args, **kwargs):
         ground_truth_state = self.annotation.ground_truth_state
         if ground_truth_state:
             self.annotation.ground_truth = ground_truth_state
-            self.annotation.save()
+            models.Model.save(self, *args, **kwargs)
```
Comment (ffont) [inline datasets/models.py]: Why this change?

Comment (xavierfav) [reply datasets/models.py]: Oh right, I though I was saving the instance itself and not doing it the right way.
Thx!

File: datasets/models.py (hunk 1 above)
Comment (xavierfav) [reply datasets/models.py]: Got it
